.github/workflows/build-and-test-all.yml

```diff
@@ -50,13 +51,18 @@ jobs:
           restore-keys: auth-ccache-
       - run: inv ci-autoconf
       - run: inv ci-auth-configure
-      - run: inv ci-auth-make-bear  # This runs under pdns/
-      - run: ln -s .clang-tidy.full .clang-tidy
+      - run: inv ci-make-distdir
+      - run: inv ci-auth-configure
+        working-directory: pdns-${{ env.BUILDER_VERSION }}
```

Comment:
Would it make sense to make that directory the default working directory for this job?

Reply:
Makes sense. I just updated the PR to include this change.

Thank you :)